Streetcode/Streetcode.BLL/Mapping/Partners/PartnerProfile.cs

```diff
@@ -13,5 +15,9 @@ public PartnerProfile()
             .ForPath(dto => dto.TargetUrl.Href, conf => conf.MapFrom(ol => ol.TargetUrl));
         CreateMap<Partner, CreatePartnerDTO>().ReverseMap();
         CreateMap<Partner, PartnerShortDTO>().ReverseMap();
+        CreateMap<StreetcodePartner, PartnersUpdateDTO>()
+            .ForPath(dto => dto.PartnerId, conf => conf.MapFrom(ol => ol.PartnerId))
+            .ForPath(dto => dto.StreetcodeId, conf => conf.MapFrom(ol => ol.StreetcodeId));
```

Comment:
it is unnecessary to set the configuration for mapping if the properties have the same name. This will reduce the number of unnecessary lines of code.